README.MD

```diff
@@ -57,8 +57,9 @@ mdbook serve
 
 Be aware of the following caveats:
 
-- `docs/html/` contains only partially processed files and it's also the folder that gets served by `mdbook serve`, so you will see some unprocessed custom markup if you visit the endpoint exposed by mdbook's development web server.
-As a workaround, you can manually serve the file in the `docs/pandocs/` with any web server (e.g. you can just run `npx http-server` in the `docs/pandocs` folder).
+- `docs/html/` contains only partially processed files and it's also the folder that gets served by `mdbook serve`, so you will see some unprocessed custom markup if you visit the endpoint exposed by mdbook's development web server (:3000).
+
+As a workaround, you can manually serve the file in the `docs/pandocs/` with the web server of your choice (e.g. you can run `python3 -m http.server` from the `docs/pandocs` folder).
```

Comment:
Careful that without two spaces of indentation, this is a paragraph that breaks the list, not a paragraph within the list item.

Reply:
Oops, you're right. I should've fixed.